src/Text/__tests__/__snapshots__/index.test.js.snap

```diff
@@ -1,27 +1,375 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Rendered with children Should contain text 1`] = `
-<React.Fragment>
-  <p
-    className="jsx-1052758011 "
+exports[`Text should match snapshot 1`] = `
+<ThemeProvider
```

Comment:
this snapshot doesn't make much sense, please keep the old one - it actually shouldn't be affected by changes in this PR (so it the test, do something like `expect(wrapper.find('Text')).toMatchSnapshot()`)

Reply:
Fixed.